cmd/snap-confine/seccomp-support.c

```diff
@@ -45,6 +47,31 @@ static const char *filter_profile_dir = "/var/lib/snapd/seccomp/bpf/";
 
 typedef struct sock_filter bpf_instr;
 
+// keep in sync with snap-seccomp/main.go
+//
+// header of a seccomp.bin2 filter file, in network byte order (big-endian)
+//
+// Note that padding is important
+// alternatively "__attribute__((__packed__))" could be used
+struct sc_seccomp_file_header {
```

Comment:
feel free to ignore this as this is definitely personal opinion, but for these kinds of structures I'd flat out use fixed-width types like `uint8_t, uint32_t` and friends. I think it's fine to keep the `header` field as `char` as it indicates we expect actual characters, but for the rest I'd keep the style the same. (i.e types from stdint.h)